libraries/redcore/api/soap/soap.php

```diff
@@ -165,7 +170,7 @@ public function apiSoap()
 		$response = ob_get_contents();
 		ob_end_clean();
 
-		$this->soapResponse = $response;
+		$this->soapResponse = str_replace('&', '&amp;', $response);
```

Comment:
it is properly parsed?
imo I would firstly go for str_replace('&amp;', '&', $response);
and then str_replace('&', '&amp;', $response); so we do not get something like &amp;amp;